### ISSUE TITLE
p:panel without header generation results in missing border-top

### ISSUE THREAD

**Issue by arnieHouston** [primefaces/primefaces#9113]
### Describe the bug

I created my own theme and that may be involved, but this is what I'm seeing. It wasn't an issue in PF 8 with one of the community themes.

If I put a p:panel around some content I expect to get a box on the page, with all four borders. What I am getting is a box with no top border. Its in the style that way (border: 1px solid #color and then border-top: 0).

Which makes sense if the panel has a header facet because the header will have a bottom border which blends into the non-existent top border.

But if no header facet is defined, there's no top border and I have to add a style to the panel to fix the issue, hard coding the style and color. If I had to change the theme again, I'd also have to search my program for all references to that code and update it, which should be avoided.

Like I said, I used to get boxes with p:panel w/o a header facet, now I don't. 

Seems like a bug to me. 

### Expected behavior

A p:panel without a header facet should render a box with all four borders.

### PrimeFaces edition

Community

### PrimeFaces version

11.0.0

### Theme

Custom

### JSF implementation

Mojarra

### JSF version

2.3

### Browser(s)

Chrome and Edge

**Comment by tandraschko** [comment]
when i run this with github master, it looks fine:

```
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml"
	xmlns:ui="http://java.sun.com/jsf/facelets"
	xmlns:f="http://java.sun.com/jsf/core"
	xmlns:p="http://primefaces.org/ui"
	xmlns:h="http://java.sun.com/jsf/html">

<h:head>
	<title>PrimeFaces Test</title>
</h:head>
<h:body>

    <p:panel>
        test test
    </p:panel>

</h:body>
</html>
```

I suggest to clone primefaces-test, run the example and adjust your own theme like the PF internal themes

**Comment by melloware** [comment]
yep and it may have been fixed since PF 8.0.
